cmd/kubelet/app/server.go

```diff
@@ -380,6 +380,11 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 		return err
 	}
 
+	kubeClient, err := getKubeClient(s)
```

Comment:
Is it possible to hook into the existing sync loop? That code already rewrites the node status based on resource info from cadvisor and OIRs.

Also looking forward to enabling operator-unregistration, we would need to get access to the updated (read) status in that sync anyway.